src/index.html

```diff
@@ -15,8 +15,99 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="#"
+      >
+        <img
+          src="./images/moyo_1.svg"
```

Comment:
Already you have image, so you need to remove these images and use prev 
<img width="346" alt="image" src="https://github.com/user-attachments/assets/66c44f1e-7863-4621-a990-aa02796564bd" />
```suggestion
          src="./images/logo.png"
```